content/docs/dvclive/api-reference/live/log_plot.md

```diff
@@ -0,0 +1,150 @@
+# Live.log_plot()
+
+```py
+def log_plot(self, name: str, labels, predictions, **kwargs):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+y_true = [0, 0, 1, 1]
+y_pred = [1, 0, 1, 0]
+y_score = [0.1, 0.4, 0.35, 0.8]]
+live.log_plot("roc", y_true, y_score)
+live.log_plot("confusion_matrix", y_true, y_pred)
+```
+
+## Description
+
+Generates a
+[scikit learn plot](https://scikit-learn.org/stable/visualizations.html) and
+saves the result in `{path}/plots/{name}.json`.
```

Comment:
Again, maybe move this to the top.

Also, "result" is vague. Maybe "data" is better to clarify that it's not the rendered plot (the json format is a big clue, but might as well be explicit).